product_docs/docs/pem/8/registering_database_server.mdx

```diff
@@ -80,9 +80,9 @@ If you don't provide the password, a password authentication error occurs. The P
 
 #### Examples
 
-The following example registers a server using only the required parameters. 
-The user `admin01` will be used to connect to the PEM server and the credentials supplied for the `--server-*` parameters will be used to create a connection from the agent to the monitored server.
-The same details will also be used to populate the server connection details in the PEM web application.
+This example registers a server using only the required parameters. 
+The user `admin01` is used to connect to the PEM server, and the credentials supplied for the `--server-*` parameters are used to create a connection from the agent to the monitored server.
+The same details are also used to populate the server connection details in the PEM web application.
```

Comment:
I'm not sure we've gained any clarity with the tense change here. To me the current wording implies "after the user presses enter, the user `admin01` will be used...". Changing to the present tense makes it less clear whether we are making a statement about what will happen as a consequence of this action, or just making a general statement about `admin01`. For example, you could read this as meaning "`Admin01` is a special user that exists for the purpose of connecting to the PEM Server"

Reply:
Okay,  I tried to come up with another way but i'm not sure it's an improvement, so I'll revert it.
```suggestion
This example registers a server using only the required parameters. 
The user `admin01` will be used to connect to the PEM server, and the credentials supplied for the `--server-*` parameters will be used to create a connection from the agent to the monitored server.
The same details will also be used to populate the server connection details in the PEM web application.
```